database/models.py

```diff
@@ -30,6 +30,7 @@ class Organization(db.Model):
 
     name = db.Column(db.String, nullable=False, index=True)
     logo = db.Column(db.String)
+    organization_type = db.Column(db.String)
```

Comment:
Do we maybe want to specify the "current" types, so as to give what typical usage of this is? Or some other type of definition?

Reply:
Now enumerated in the latest commit.